internal/cmd/config.go

```diff
@@ -303,3 +335,34 @@ func validateScenarioConfig(conf lib.ExecutorConfig, isExecutable func(string) b
 	}
 	return nil
 }
+
+// migrateLegacyConfigFileIfAny moves the configuration file from
+// the old default `~/.config/loadimpact/...` folder
+// to the new `~/.config/k6/...` default folder.
+func migrateLegacyConfigFileIfAny(gs *state.GlobalState) error {
```

Comment:
Can this be a part of `readDiskConfig` (e.g., calling this function from readDiskConfig)? Do we need to know the details of how we're getting the config file?